tests/test_automation/jenkins_rhea_cpu.sh

```diff
@@ -30,7 +30,7 @@ mkdir -p build
 
 cd build
 
-# real, full precision
+echo "starting new test for real full precision"
```

Comment:
Could you please add a few empty lines by echo if there is no blank lines between this message and the previous tests?

Reply:
There are blank lines after the final unit tests run.